apps/docs/content/guides/auth/auth-smtp.mdx

```diff
@@ -1,38 +1,136 @@
 ---
 id: 'auth-smtp'
-title: 'Configure a Custom SMTP'
+title: 'Send emails with custom SMTP'
 description: 'Moving towards production: Configuring a custom SMTP provider'
 ---
 
-## Auth SMTP
+If you're using Supabase Auth with the following configuration:
 
-At present, you can trial the Supabase platform by sending up to **3** emails per hour via the built-in service. The default email service as a whole is offered on a best effort basis: we will do our best to maintain it and will review usage of the service on a regular basis to see if the email service should be continued.
+- Email and password accounts
+- Passwordless accounts using one-time passwords or links sent over email (OTP, magic link, invites)
+- Email-based account invitations from the Supabase dashboard or from the Auth admin APIs
```

Comment:
Can consider highlighting that this is for the project invites and that Organizational invites are not affected. Might not be evident for new-to-supabase users